tg-bot/src/main/java/com/tutrit/tgbot/example/repository/RepositoryUserEmployee.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.tgbot.example.repository;
+
+import com.tutrit.tgbot.example.bean.UserEmployee;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class RepositoryUserEmployee {
```

Comment:
When 2/3 of array limit reached, make a copy of array and increase capacity in 1.5 times